functions/tsconfig.json

```diff
@@ -1,19 +1,13 @@
 {
+  "extends": "../tsconfig.json",
   "compilerOptions": {
-    "esModuleInterop": true,
+    "baseUrl": "functions",
+    "composite": false,
     "incremental": true,
-    "lib": ["DOM", "DOM.Iterable", "ESNext"],
-    "moduleResolution": "node",
-    "module": "esnext",
-    "noEmit": true,
-    "strict": true,
```

Comment:
removing strict?

Reply:
strict is enabled in the `tsconfig` we're extending so still using strict, just don't need to redeclare it